CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 2019-07-04
+
+10.2.1 update:
```

Comment:
Drop this line, that breaks the standard format.

Reply:
done